### PR TITLE
nicolette - masterpiece of a poem added

### PR DESCRIPTION
**Pull request by nvere**
Added a poem

### CODE REVIEW

**Comment by kameronseabrook** [comment]
This is great!

**Review by kameronseabrook — COMMENTED** [review]
Works!

**Review by kameronseabrook — APPROVED** [review]
Cool coffee cup

File: script.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ const poems = [
     poem: "Roses are red \n Violets are blue \n Jerry Seinfeld is considering \n Making Bee Movie 2",
     image: "bee.jpeg",
   },  
+  {
+    author: "Nicolette",
+    poem: "Roses are red \n I am tired \n Two cups of coffee before noon \n Was a mistake",
+    image: "coffee.jpg",
+  }
```
Comment (andrewgsalmon) [inline script.js:21]: yuuuuuup!

File: styles.css [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 /* YOUR CODE HERE: Change the background-color property of the <body> tag to a different color of your choice. */
 body {
-  background-color: red;
+  background-color: black;
```
Comment (andrewgsalmon) [inline styles.css:9]: Yesss!

**Review by andrewgsalmon — APPROVED** [review]
Good!

**Review by Nicholos-Steadman — APPROVED** [review]

**Pull request merged**